deploy/config/README.md

```diff
@@ -0,0 +1,5 @@
+# Wikibase Suite Deployment Kit Configuration
+
+Will directory will contain your configuration files after the first run of the Wikibase Suite Deployment Kit. Feel free to make adjustments as you see fit. Backup this folder regularly in order to restore your configuration later.
```

Comment:
"Will directory will contain" to "Directory wil contain"

Reply:
Thanks, fixed